tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/LcovPrinter.java

```diff
@@ -38,30 +38,18 @@ private LcovPrinter(BufferedWriter bufferedWriter) {
     this.bufferedWriter = bufferedWriter;
   }
 
-  static boolean print(OutputStream outputStream, Coverage coverage) {
-    BufferedWriter bufferedWriter;
-    try (Writer fileWriter = new OutputStreamWriter(outputStream, UTF_8)) {
-      bufferedWriter = new BufferedWriter(fileWriter);
-      LcovPrinter lcovPrinter = new LcovPrinter(bufferedWriter);
-      lcovPrinter.print(coverage);
-      bufferedWriter.close();
-    } catch (IOException exception) {
-      logger.log(Level.SEVERE, "Could not write to output file.");
-      return false;
-    }
-    return true;
+  static void print(OutputStream outputStream, Coverage coverage) throws IOException {
+    Writer fileWriter = new OutputStreamWriter(outputStream, UTF_8);
```

Comment:
Should still keep the try-with-resource (or try-finally) to close the writer.

Reply:
Thanks, my java is rusty.  Added buffered writer in as well